### DISCUSSION TITLE
Have some way to bring together buttons that aren't room specific, such as heating, vacuuming, etc.

### DISCUSSION THREAD

**Discussion by douginoz** [dwainscheeren/dwains-lovelace-dashboard#714] [Ideas]
Right now I have to either place my thermostat in a specific room, or create a room that isn't a room and place things in it.  That seems like a kludge and goes against the architectural design of DD.
Things like heating, temperature probes around the house, vacuuming maps etc. which aren't room-specific should be shown in one logical place.
There's sort of a place below the "Good Morning" and current weather, which shows a row of icons for people, total # of lights on, etc. but this isn't configurable.  What would work well is to be able to add additional buttons that when pressed pops up or changes the display to that function.
Conceptually these things are "home functions" I suppose.  Automations, devices, displays etc. that don't belong in a single room.

Currently, the design of DD forces me to choose a room to place things in and it's then up to the rest of the household to remember where I put it.  Are the vacuum controls in the living room, or the hallway?  Forcing the user to associate the physical location of the robot vacuums is poor design because the use of the vacuum has nothing to do with where it's physically located.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
